test/helpers/aws.go

```diff
@@ -1,4 +1,4 @@
-//go:build aws
+//go:build aws || awsgcp
```

Comment:
NIT: Do we need to also run the awsgcp tests here? Seems like a waste of resources, given that we're going to run `aws` tests on their own anyways.

Reply:
We need access to aws and gcp test files to access helper functions such as delete and create bucket.
But the tests themselves will not be launched because we filter their list 
https://github.com/gruntwork-io/terragrunt/blob/c88754cf894d6fd24472ec66f0b5f5dde2ec5476/.circleci/config.yml#L106